ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -57,8 +57,11 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
+  commands: make community
```

Comment:
My knowledge of the release repo is lacking -- will this execute before the step `windows-e2e-operator-test`?

Reply:
I looked into this more - I think we follow a workflow/multistage layout so added a pre chain to run make community-bundle before windows-e2e-operator-test runs. 
Ref: https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow